Update README, split into USAGE.md and INSTALL.md (#1199)
confluentinc/vscode: docs/DOCUMENTATION.md

### Bootstrap streaming projects from Confluent-provided templates

<!-- <gif showing clicking button, clicking on a template, fill in values, opening the project, and running it> -->

Confluent for VS Code offers project templates designed to accelerate your development process. These ready-to-use templates are tailored for common development patterns, allowing you to quickly launch new projects with minimal configuration.

> [!NOTE]
> We will continue to add new templates as well as improving existing ones, however, please [file an issue]() or [start a discussion]() if you find bugs/issues in existing templates or have proposals for new templates that could benefit the broader community.

### Accelerate local development against Apache Kafka® and Confluent Schema Registry using Docker

> [!NOTE]
> Pre-requisites: You must have Docker Engine (version v24.0 or later) installed on your machine. Please follow the instruction [here](https://docs.docker.com/engine/install/) to install Docker Engine.

Using just a single click, you can easily start a Kafka and Confluent Schema Registry container on your
local machine (via Docker).

> 1. Click the green play button to begin the Local resources quickpick flow
> 1. Choose whether you want to create just a Kafka cluster or Kafka and Schema Registry, both

Reviewer: ```suggestion
1. Click the green play button to choose whether you want to start Kafka, or Kafka and Schema Registry together
```
What do you think about combining these two items?
